src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] machines = new Machine[3];
+        machines[0] = new Truck();
+        machines[1] = new Excavator();
+        machines[2] = new Bulldozer();
```

Comment:
Consider using abstract references instead of specific ones for the array of `Machine` instances. This will allow you to use polymorphism more effectively. For example: `Machine machine1 = new Truck();`